app/lib/menu/menu-builder.js

```diff
@@ -8,6 +8,9 @@
  * except in compliance with the MIT License.
  */
 
+/* eslint-disable no-mixed-spaces-and-tabs */
+
+
```

Comment:
Why is this needed? I'd rather fix the linting issues instead of ignoring them.

Reply:
@pinussilvestrus okay I am looking into this! Anything else that needs to be changed??

Reply:
@pinussilvestrus I have solved the issue :)